codalab/model/worker_model.py

```diff
@@ -209,6 +209,10 @@ def get_stream(self, sock, timeout_secs):
         sock.settimeout(timeout_secs)
         try:
             conn, _ = sock.accept()
+            # Send Ack. This helps protect from messages to the worker being
+            # lost due to spuriously accepted connections when the socket
+            # file is deleted.
+            conn.sendall('a')
```

Comment:
Can you define `ACK = 'a'` and replace `1` with `len(ACK)` below to more explicitly connect these two parts of the code semantically?